core/src/main/java/org/apache/brooklyn/enricher/stock/Aggregator.java

```diff
@@ -64,6 +65,10 @@
     
     public static final ConfigKey<Boolean> EXCLUDE_BLANK = ConfigKeys.newBooleanConfigKey("enricher.aggregator.excludeBlank", "Whether explicit nulls or blank strings should be excluded (default false); this only applies if no value filter set", false);
 
+    public static final ConfigKey<String> QUORUM_CHECK_TYPE = ConfigKeys.newStringConfigKey("quorum.check.type", "The requirement to be considered quorate", "allAndAtLeastOne");
```

Comment:
Worth indicating the possible values in the description? Appreciate there's a chance it could get out of sync with the strings in QuorumCheck.